modin/core/dataframe/pandas/partitioning/partition_manager.py

```diff
@@ -1148,6 +1148,30 @@ def _apply_func_to_list_of_partitions(cls, func, partitions, **kwargs):
         preprocessed_func = cls.preprocess_func(func)
         return [obj.apply(preprocessed_func, **kwargs) for obj in partitions]
 
+    @classmethod
+    def convert_partitions_to_pandas_in_remote_func(cls, partitions):
+        """
+        Call to_pandas for the given NumPy array of partitions.
+
+        For a NumPy array of partitions create a single partition
+        that has the data for all partitions.
+
+        Parameters
+        ----------
+        partitions : np.ndarray
+            The partitions which have to be converted to a single partition.
+
+        Returns
+        -------
+        PandasDataframePartition
+            A PandasDataframePartition object which holds the data of all partitions.
+        """
+
+        def call_to_pandas(data, partitions):
```

Comment:
Maybe it can help to fix tests on Ray in CI:
```suggestion
        cls.finalize(partitions)
        def call_to_pandas(data, partitions):
```